extensions/Lily/SoundExpanded.js

```diff
@@ -24,13 +24,43 @@
             opcode: "startLooping",
             blockType: Scratch.BlockType.COMMAND,
             text: "start looping [SOUND]",
+            arguments: {
+              SOUND: {
+                type: Scratch.ArgumentType.SOUND,
+              },
+            },
+            extensions: ["colours_sounds"],
+          },
+          {
+            opcode: "startLoopingBegin",
+            blockType: Scratch.BlockType.COMMAND,
+            text: "start looping [SOUND] begin [START]s",

```

Comment:
change "s" to "secs", and this is perfect

Reply:
I would say "seconds" for parity